### ISSUE TITLE
[FR] Improve `nitro iniset` boolean prompts

### ISSUE THREAD

**Issue by mattstein** [craftcms/nitro#361]
### Description

Nitro’s `iniset` command prompts with a question for the desired setting, and its auto-suggestions for boolean values are `true` or `false`.

Example:

```
$ nitro iniset europa.nitro
Which PHP setting would you like to change for europa.nitro?
  1. display_errors
  2. max_execution_time
  3. max_input_vars
  4. max_input_time
  5. max_file_upload
  6. memory_limit
  7. opcache_enable
  8. opcache_revalidate_freq
  9. opcache_validate_timestamps
  10. post_max_size
  11. upload_max_file_size
Enter your selection: 7
Should we enable OPCache [false]?
```

These should be `Y/n` answers because:

1. We’re answering a “should we” question, and it’s more natural for humans to answer “yes” or “no”.
2. It would be more consistent with the rest of Nitro’s `Y/n` prompts.
3. It reduces input effort to typing a single character instead of four or five—unless you’re accepting the default, which is not always a sensible suggestion.

They also appear to suggest the PHP default, which means you can indicate you’d like to change a setting and be nudged toward accepting the same one that’s already active. Ideally, the prompt would recommend the opposite of the current value since the user already indicated they’d like to change it.

### tl;dr

- Prompts for booleans should prompt for `Y` or `n`.
- Prompts for booleans should auto-suggest the opposite of the current value, or get rid of auto-suggestions if they don’t make sense in the context of the previous selection.

**Comment by jasonmccallister** [comment]
Hi, we are closing this issue as we have decided to retire Nitro, so no additional work will occur on this project. You can read the official blog post here https://craftcms.com/blog/retiring-craft-nitro. We appreciate everyones feedback and involvement and we look forward to refocusing our efforts on Cloud! 

If you're looking for a new local development environment, we recommend DDEV and have a knowledge base article to help you with the transition: https://craftcms.com/knowledge-base/migrating-from-craft-nitro-to-ddev.
